back/app/services/side_fx_idea_service.rb

```diff
@@ -105,6 +120,23 @@ def create_followers(idea, user)
 
     Follower.find_or_create_by(followable: idea.project.folder, user: user)
   end
+
+  def sanitize_location_point(change)
+    return change if change['location_point'].blank?
+
+    geojson_change = change['location_point'].map { |point| RGeo::GeoJSON.encode(point) }
+
+    # We encode the location_point to GeoJSON and then check for changes, else it would always be considered changed
+    # when a location_point was set, due to the fact that the location_point previous change is encoded as
+    # RGeo::Geographic::SphericalPointImpl, while the current change is encoded as RGeo::Geos::CAPIPointImpl.
+    if geojson_change[0] == geojson_change[1]
+      change.delete('location_point')
+    else
+      change['location_point'] = geojson_change
+    end
+
+    change
+  end
```

Comment:
This seems a bit weird, and I think ideally I would find a way to avoid the changing `RGeo` type in `saved_changes`. Hopefully, I will have time to look into this, but in the meantime this seems to handle the issue.